target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -157,6 +157,7 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
+	comfast,cf-e375ac|\
```

Comment:
No MAC address in 0x5006 although there is one in 0x1002?

Reply:
There's indeed a valid address in 50006 but vendor firmware won't assign it to any interface